ocean_lib/models/test/test_datatoken.py

```diff
@@ -33,13 +35,48 @@ def test_ERC20(alice_ocean, alice_wallet, alice_address, bob_wallet, bob_address
     assert from_base_18(token.balanceOf(alice_address)) == 98.0
     assert from_base_18(token.balanceOf(bob_address)) == 2.0
 
+    # test status functions
+    assert token.totalSupply() == 100_000_000_000_000_000_000
+    assert token.cap() == 1_000_000_000_000_000_000_000
+    assert token.datatoken_name() == "DataToken1"
+    block = alice_ocean.web3.eth.blockNumber
+    token_info = token.get_info(
```

Comment:
The name of the test is test_ERC20. To test ERC20 functionality.

get_info() is not ERC20 compliant. So it shouldn't be part of "test_ERC20", make it a new test.

BTW I just realized that a bunch of ocean.py datatoken interface isn't compliant :/ So I just created an issue for it. https://github.com/oceanprotocol/ocean.py/issues/228

For the purposes of _this_ PR, we don't have to fix that issue of course. To handle this comment, simply create another test method like "def test_get_info" and move its stuff there.